requestor/worldstate.go

```diff
@@ -141,3 +144,8 @@ func DailyDeals(platform string) {
 		fmt.Println("The information could not be fetched")
 	}
 }
+
+func printNumberOfRecordsFetched(resultArrayLen int) {
+	var numberOfRecords string = fmt.Sprintf("%d", resultArrayLen)
+	fmt.Println("\n" + colorformat.ForeGrnBold("Total number of records fetched: "+numberOfRecords) + "\n")
+}
```

Comment:
Try type-casting the integer directly to string using `string(number)`. Also, I do not think we would need a separate function for operations as small as these.

Reply:
Sure, I would do the requested changes

Reply:
And for casting integer to string, should I use strconv?

Reply:
I would abstain from using `strconv` for it adds some overhead. The overhead is small but still noticeable.

Reply:
I did the changes you requested for, although, using string(number) conversion, gave me some weird results in the terminal. (see for  --drvodeal)
![Screenshot from 2020-10-17 16-40-50](https://user-images.githubusercontent.com/20144724/96335632-b7853a80-1097-11eb-8497-4d5eac897338.png)

Reply:
> I did the changes you requested for, although, using string(number) conversion, gave me some weird results in the terminal. (see for --drvodeal)

Umm. Right, that is leading to issues here too. Please stick to `strconv` then.

Reply:
Pushed the changes with strconv refactor